notificationworkerlambda/src/main/scala/com/gu/notifications/worker/Configuration.scala

```diff
@@ -46,79 +41,81 @@ case class TopicCountsConfiguration (
 
 object Configuration {
 
-  private def fetchConfiguration(): Config = {
+  private def fetchConfiguration(workerName: String): Config = {
     val identity = AppIdentity.whoAmI(defaultAppName = "notification-worker")
     ConfigurationLoader.load(identity) {
-      case AwsIdentity(_, _, stage, _) => SSMConfigurationLocation(s"/notifications/$stage/workers")
+      case AwsIdentity(_, _, stage, _) => SSMConfigurationLocation(s"/notifications/$stage/workers/$workerName")
     }
   }
 
   def platform: Option[Platform] = Option(System.getenv("Platform")).flatMap(Platform.fromString)
 
+  def confPrefixFromPlatform: String = platform match {
+    case Some(p @ (Ios | Android | IosEdition | AndroidEdition)) => p.toString
+    case _ => throw new IllegalStateException("No Platform environment variable defined")
+  }
+
   private def jdbcConfig(config: Config) = JdbcConfig(
     driverClassName = "org.postgresql.Driver",
     url = config.getString("registration.db.url"),
     user = config.getString("registration.db.user"),
     password = config.getString("registration.db.password"),
-    maxConnectionPoolSize = config.getInt("registration.db.maxConnectionPoolSize")
+    maxConnectionPoolSize = 1
   )
 
+  def fetchHarvester(): HarvesterConfiguration = {
+    val config = fetchConfiguration("harvester")
+    HarvesterConfiguration(
+      jdbcConfig = jdbcConfig(config),
+      iosLiveSqsUrl = config.getString("iosLiveSqsUrl"),
+      iosEditionSqsUrl = config.getString("iosEditionSqsUrl"),
+      androidLiveSqsUrl = config.getString("androidLiveSqsUrl"),
+      androidEditionSqsUrl = config.getString("androidEditionSqsUrl")
+    )
+  }
+
   def fetchApns(): ApnsWorkerConfiguration = {
-    val config = fetchConfiguration()
+    val config = fetchConfiguration(confPrefixFromPlatform)
     ApnsWorkerConfiguration(
-      jdbcConfig(config),
-      config.getString("cleaner.sqsUrl"),
+      config.getString("cleaningSqsUrl"),
       ApnsConfig(
         teamId = config.getString("apns.teamId"),
-        bundleId = if (platform.contains(Ios)) config.getString("apns.bundleId") else config.getString("apns.newsstandBundleId"),
+        bundleId = config.getString("apns.bundleId"),
```

Comment:
Here I use to test what platform we were using (ios or newsstand?) to check which property to load. But now the configuration is prefixed and each platform has its own version of the config.